packages/shared/src/components/modals/squads/PostModerationModal.tsx

```diff
@@ -67,7 +68,7 @@ function PostModerationModal({
       size={Modal.Size.Large}
       kind={Modal.Kind.FlexibleTop}
     >
-      <Modal.Body className="gap-6">
+      <Modal.Body className="gap-6 !p-6">
```

Comment:
Can we not override the padding without using important? I think we should avoid it whenever necessary

Reply:
Unfortunately this is not possible due to the order in which the classes are written by TailwindCSS. 
We have discussed this issue internally in the past and proposed using [twMerge](https://www.npmjs.com/package/tailwind-merge) but it has not been adopted yet :(

Reply:
For this it's fine, ideally all modals follow guidelines, but this is a weird modal I guess.